cluster/crds/pkg.crossplane.io_configurationrevisions.yaml

```diff
@@ -126,6 +126,14 @@ spec:
                   whether to skip resolving dependencies for a package. Setting this
                   value to true may have unintended consequences. Default is false.
                 type: boolean
+              tlsClientSecretName:
+                description: TLSClientSecretName is the name of the TLS Secret that
+                  stores client certificates of the Provider to call Functions.
+                type: string
+              tlsServerSecretName:
+                description: TLSServerSecretName is the name of the TLS Secret that
+                  stores server certificates of the Provider.
+                type: string
```

Comment:
doesn't make much sense for configurations 🤔 would there be a way not to have them added here?

Reply:
I agree, but I couldn't find a way to do that.